wallet/src/testing.rs

```diff
@@ -232,6 +242,27 @@ pub async fn mint_token<'a>(
     }
     Ok(my_asset)
 }
+
+/// Return records the freezer has access to freeze or unfreeze but does not own.
+/// Will only return records with freeze_flag the same as the frozen arg.
+pub async fn find_freezable_records<'a>(
+    freezer: &CapeWallet<'a, CapeBackend<'a, ()>>,
+    frozen: FreezeFlag,
+) -> Vec<RecordInfo> {
+    let pks: HashSet<UserPubKey> = freezer.pub_keys().await.into_iter().collect();
```

Comment:
This should be `let pks: HashSet<FreezerPubKey> = freezer.freezer_pub_keys().await.into_iter().collect();`